scripts/commit-build.sh

```diff
@@ -72,7 +73,9 @@ commit_build() {
 # Is the build asking to be committed to test?
 # ======================================================
 if [[ "$TRAVIS_BRANCH" == "$STACHE_DEVELOP_BRANCH" ]]; then
-  commit_build $STACHE_DEPLOY_TEST_BRANCH
+  if [[ $LAST_COMMIT_MESSAGE =~ $REGEX_DEPLOY_TEST_COMMENT ]]; then
```

Comment:
This makes it so that only merged PRs trigger a deployment? Adds more structure around the test branch so it mimics the prod. I like it.